internal/convert/api_keys_access_list.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package convert
```

Comment:
[nb] Not a particular fan of making these methods public, and adding them to this package but for now I think is ok

Reply:
Could you tell me why? I see other functions that are public in this package

Reply:
This package was intended only for the automation config, it now has database users logic, and we are now adding access list logic as well, I've been wanting to take the db logic from here for a while but it's a bit reused in a couple of places so it's a bit harder to do. for this access list given how is only used for the store, and should only belong to the store logic they could be private methods there

Reply:
Done